server/src/test/java/org/elasticsearch/discovery/AbstractDisruptionTestCase.java

```diff
@@ -93,6 +94,11 @@ public void setUp() throws Exception {
         disableBeforeIndexDeletion = false;
     }
 
+    @After
+    public void assertSameDocIds() throws Exception {
+        assertSameDocIdsOnShards();
```

Comment:
There's already a method called `assertSeqNos`. Should we call that one first? can you also check `assertSameDocIdsOnShards` for RelocationIT?